src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -186,14 +186,20 @@ Supported in Sentry Android Gradle Plugin version `3.0.0` and above.
 
 </Note>
 
-The [Sentry Android Gradle Plugin](/platforms/android/proguard/#gradle-configuration) does the tracing auto instrumentation via bytecode manipulation for `SQLite` and `Room` libraries.
+The [Sentry Android Gradle Plugin](/platforms/android/proguard/#gradle-configuration) does the tracing auto instrumentation via bytecode manipulation for `androidx.sqlite` and `androidx.room` libraries.
 
 The Plugin injects a code snippet that starts a span out of the active span bound to the scope for each `CRUD` operation. The SDK sets the span `operation` to `db` and `description` to the SQL Query if available.
 
 The span finishes once the operation has been executed. The span `status` is set to `SpanStatus.OK` if successful or `SpanStatus.INTERNAL_ERROR` if there was any error.
 
 When the operation throws an `Exception`, Sentry's SDK associates this exception to the running span. If you haven't set the SDK to swallow the exception and capture it, the span and SentryEvent will be linked when viewing it on the **Issue Details** page in sentry.io.
 
+<Note>
+ 
+At the moment, only a standard `androidx.room` usage is supported, i.e. SQL queries for any other `SupportSQLiteOpenHelper.Factory` than [androidx.sqlite](https://github.com/androidx/androidx/tree/androidx-main/sqlite) will not be reported. However, it's worth reporting an [issue](https://github.com/getsentry/sentry-android-gradle-plugin/issues), as we could consider adding a new integration.
+
+</Note>
```

Comment:
looks like the code highlight is not working for `Note`, See https://sentry-docs-git-enh-sqlite-instrumentation-clarificaiton.sentry.dev/platforms/android/performance/instrumentation/automatic-instrumentation/

Reply:
no idea, looks correct.

Reply:
I'm also stumped. Once we're done with the wording changes, I can pull in your branch and try to play with it.

Reply:
Maybe it's not supported in 2 adjacent Notes? not sure lol